ios/samples/app-template.yml

```diff
@@ -17,25 +17,27 @@ targetTemplates:
             configVariants:
                 - Metal
                 - OpenGL
+        dependencies:            
+            - framework: "../../out/ios-release/filament/lib/libbackend.xcframework"
```

Comment:
We don't want to hardcode this to `ios-release`, as then this won't work with debug builds. I think we might need to move this under `settings.config.release` and modify it for `settings.config.debug `

Reply:
Ah yes that must be a different type of framework path. Confusing!